logger/sql.go

```diff
@@ -13,10 +13,12 @@ import (
 	"gorm.io/gorm/utils"
 )
 
+// TimeParamFormat defines the format of the time parameter in ExplainSQL.
+var TimeParamFormat = "2006-01-02 15:04:05.999"
+
 const (
-	tmFmtWithMS = "2006-01-02 15:04:05.999"
-	tmFmtZero   = "0000-00-00 00:00:00"
-	nullStr     = "NULL"
+	tmFmtZero = "0000-00-00 00:00:00"
```

Comment:
> What do you think about using the same tmFmtZero and TimeParamFormat format? IMO This would make confusion later if the TimeParamFormat is not used for zero values.

@saeidee 
Thank you for your comment.  I have considered your suggestion, but using the same `TimeParamFormat` for zero time values would lead to a breaking change, so I have maintained the current behavior for now.

```
before: 0000-00-00 00:00:00
after:  0001-01-01 00:00:00
```